midterm-2023-11-07/code/5040-dara-poposka/index.html

```diff
@@ -1,49 +1,61 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>MidTerm Exam for Internet Programming</title>
-
-    <link rel="stylesheet" href="styles.css">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>List of Movies</title>
+  <link rel="stylesheet" href="styles.css">
 </head>
 <body>
-    <h1>Movies Information</h1>
-    <div class="movie-header-table">
-      <div class="movie-header">ID</div>
-      <div class="movie-header">Title</div>
-      <div class="movie-header">Director</div>
-      <div class="movie-header">Year</div>
-      <div class="movie-header">Genre</div>
-      <div class="movie-header">Plot</div>
-      <div class="movie-header">Cast</div>
-      <div class="movie-header">Oscars</div>
+  <header>
+    <h1>List of Movies</h1>
+    <div id="filters" class="filter">
+      <label for="title">Title:</label>
```

Comment:
The `for` text, should be an ID attribute of an input element, so the correct value is `title-filter`